server/server_test.go

```diff
@@ -159,7 +159,7 @@ func TestOAuth2CodeFlow(t *testing.T) {
 	clientSecret := "testclientsecret"
 	requestedScopes := []string{oidc.ScopeOpenID, "email", "profile", "groups", "offline_access"}
 
-	t0 := time.Now()
+	t0 := time.Now().UTC()
```

Comment:
This value gets passed on to the now() func which gets used in newKeyCacher(). So this flow would mimic the actual behavior of the server now() function.